shoop/simple_pricing/migrations/0002_remove_simpleproductprice_includes_tax.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('simple_pricing', '0001_initial'),
```

Comment:
I wonder if this should actually depend on the core migration adding the per-shop taxation field.

Then we could have a data migration that toggles the per-shop taxation field based on the current taxness of the prices for each shop. (Ambiguous cases, where a shop has both taxful and taxless prices, would warrant a warning.) Otherwise we're just, well, dropping taxfulness information here.